Dockerfile.server

```diff
@@ -56,12 +56,12 @@ RUN BUILD_SEQUENCIAL=1 yarn install --inline-builds \
 FROM node:16-bullseye-slim
 
 ARG user=joplin
-RUN useradd --create-home --shell /bin/bash $user
+RUN usermod -d /home/joplin -l joplin node
 
 USER $user
 
-COPY --chown=$user:$user --from=builder /build/packages /home/$user/packages
-COPY --chown=$user:$user --from=builder /tini /usr/local/bin/tini
+COPY --chown=$user:node --from=builder /build/packages /home/$user/packages
+COPY --chown=$user:node --from=builder /tini /usr/local/bin/tini
```

Comment:
I don't understand what is going on here. First you rename the `node` user to `joplin` (hardcoded) and then you change the group owner to `node`.

Reply:
If using the user setting the volume's own is node

Reply:
I think we are talking past each other. I know what chown does. 

Look at line 59: `RUN usermod -d /home/joplin -l joplin node` is the problem.